src/babel/transformation/file/option-parsers.js

```diff
@@ -26,3 +26,11 @@ export function booleanString(key, val) {
 export function list(key, val) {
   return util.list(val);
 }
+
+// We have to use abbreviated `func` because
+// `function function() {}` is a syntax error
+export function func(key, val) {
+  // Using an array as a trick to capture anonymous
+  // functions, which would otherwise error
+  return eval(`[${val}]`)[0];
```

Comment:
I'm open to suggestions on alternative methods...Not a huge fan of `eval`'ing command line args for obvious reasons, but I can't honestly think of any other sane approach.